src/pages/home/report/ReportActionCompose.js

```diff
@@ -289,6 +338,44 @@ class ReportActionCompose extends React.Component {
                             </>
                         )}
                     </AttachmentModal>
+                    <PopoverWithMeasuredContent
+                        isVisible={this.state.isPickerVisible}
+                        onClose={this.hideEmojiPicker}
+                        anchorPosition={this.popoverAnchorPosition}
+                        animationIn="fadeIn"
+                        anchorOrigin={
+                            {
+                                horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT,
+                                vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                            }
+                        }
+                        measureContent={() => (
+                            <EmojiPickerMenu
```

Comment:
Up to you if you want to handle the refactoring, but even though I created this component I've realized after-the-fact that the `measureContent` component generally isn't necessary, so I'd prefer to see the `PopoverWithMeasuredContent` measure it's children by defaut.

Reply:
Updated `PopoverWithMeasuredContent` let me know if it's not what you meant!